code/__DEFINES/atmospherics/atmos_piping.dm

```diff
@@ -37,7 +37,7 @@
 #define QUATERNARY 4
 
 //TANKS
-/// The volume of the standard handheld gas tanks on the station.
+/// The volume of the standard handheld gas tanks on the station. Around ~17.75 moles.
```

Comment:
I think this comment change needs a little more detail. Volume and number of mols measure two different things; volume is the amount of space for the gas to occupy, mols is the amount of gas present. A 70 liter tank will not necessarily have 17.75 mols in it; it depends on the pressure and temperature as well. The pre-spawned tanks that are at 20 C and about 600 kpa will have 17.75 mols. You should clarify the comment to include that.

Reply:
okay nerd